### DISCUSSION TITLE
tket repository size

### DISCUSSION THREAD

**Discussion by 1tnguyen** [CQCL/tket#756] [General]
I'm wondering why the repository size is so large (~1.54 GB). It might take a couple of minutes to perform a fresh clone.

**Comment by cqc-alec** [comment]
The reason is the size of the docs, coverage reports etc. on the `gh-pages` branch, which are frequently updated and very large. We are aware of the problem and are looking to find a way around it (probably by using the `deploy-pages` action instead, and compressing the history of that branch).
